### ISSUE TITLE
Feature request: option to hide the icon in the menu bar

### ISSUE THREAD

**Issue by Soares** [beardedspice/beardedspice#169]
I am averse to having many icons in my menu bar :-)

**Comment by blazarecki** [comment]
+1

**Comment by micros123** [comment]
Did you try the free app Bartender? You can choose which icons you want to hide.

**Comment by blazarecki** [comment]
Yes, but I think having an option in the app could be a great feature. And maybe can choose between several icon. What do you think ?

**Comment by davidneedham** [comment]
+1. This would be convenient.

FYI: [Bartender](https://www.macbartender.com/) does look handy, but it's $15. A free alternative for hiding and reordering menu icons is [Broomstick](http://www.zibity.com/goodies.html), but it doesn't yet support BeardedSpice.

**Comment by Ch1ll3r** [comment]
+1 with a notch, you want to have the least amount of menu bar icons possible.

**Comment by vityevato** [comment]
@Ch1ll3r https://github.com/Stillness-2/beardie
